pages/index.js

```diff
@@ -0,0 +1,78 @@
+import Head from 'next/head';
+import Header from '../components/Header.js';
+import Footer from '../components/Footer.js';
+import React from 'react';
+
+export default function Home() {
+  return (
+    <div className="container">
+      <Head>
+        <title>Bist du Teppich </title>
+        <link rel="icon" href="/favicon.png" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=DM+Mono&display=swap"
+          rel="stylesheet"
+        ></link>
+      </Head>
+      <Header />
+      <main>
+        <h1 className="title"> Welcome to Bist du Teppich!</h1>
+        <img alt="item" src="/favicon.png"></img>
+        <h2>Home of most cosy interior decor</h2>
+      </main>
+      <Footer />
+
+      <style jsx>{`
+        .container {
+          margin-top: 70px;
+          min-height: 100vh;
+        }
+
+        img {
+          margin-top: -40px;
+          height: 300px;
+        }
+
+        main {
+          padding: 0;
+
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: center;
+        }
+
+        .title {
+          color: #2f3640;
+          text-decoration: none;
+          text-shadow: 2px 2px black;
+          margin-top: 70px;
+          line-height: 1.15;
+          font-size: 4rem;
+          text-align: center;
+        }
+
+        h2 {
+          font-size: 2rem;
+          text-shadow: 1.5px 1.5px black;
+          margin-top: -20px;
+        }
+      `}</style>
+
+      <style jsx global>{`
```

Comment:
You can do global things like this that you want to load on every page in a special, separate component (very often called `<Layout />`)